package.json

```diff
@@ -12,7 +12,7 @@
     "deploy": "script/deploy",
     "lint": "eslint ./ --ignore-path .gitignore",
     "lint-fix": "npm run lint -- --fix",
-    "precommit": "lint-staged",
+    "precommit": "lint-staged && npm test",
```

Comment:
This is cool, but I wanted to flag.

It can suck, because if you just want to commit some broken stuff to save it, it will stop you.

That said, if you just run the command:
`git commit --no-verify`

And it will skip those checks.